platform-mbean/src/main/resources/org/jboss/as/platform/mbean/LocalDescriptions.properties

```diff
@@ -191,6 +191,8 @@ threading.dump-all-threads.suspended=Whether the thread is suspended.
 threading.dump-all-threads.in-native=Whether the thread associated is executing native code via the Java Native Interface (JNI).
 threading.dump-all-threads.locked-monitors=A list of detyped representations java.lang.management.MonitorInfo objects, each of which represents an object monitor currently locked by the thread.
 threading.dump-all-threads.locked-synchronizers=A list of detyped representations java.lang.management.LockInfo objects, each of which represents an ownable synchronizer currently locked by the thread.
+threading.dump-all-threads.daemon=Tests if the thread associated with this ThreadInfo is a daemon thread.
```

Comment:
"Whether the thread is a daemon thread."

^^^ is to make the style of this consistent with the other fields.